docs/developers/protocol/hermez-protocol/protocol.md

```diff
@@ -632,16 +630,16 @@ Sender sends the transaction to a Baby Jubjub address recipient in the state tre
 If the recipient does not exist and coordinator wants to process the transaction, it should create a new account with the recipient Baby Jubjub address.
 
 It is assumed that the `toIdx` is set to the special index 0.
-`toBjjAy` + `toBjjSign` would be used to choose the recipeint to transfer the `amountFloat16`.
+`toBjjAy` + `toBjjSign` would be used to choose the recipeint to transfer the `amountFloat40`.
 `toEthAddr` will be set to `0xff..fff` which is a special case of an Ethereum address that no one can control and it will be check that the recipient account has its Ethereum address set to `0xff..ff`.  This value allows account creation without Ethereum address authorization.
-Hence, coordinator would select the recipient `idx` to add `amountFloat16` (called `auxToIdx`).
+Hence, coordinator would select the recipient `idx` to add `amountFloat40` (called `auxToIdx`).
 
 > Note that this transaction encourage coordinador to create new accounts through L1 coordinator transaction [CreateAccountBjj](#createaccountbjj).
```

Comment:
Note that this transaction encourages the coordinator to create new accounts through the L1 coordinator transaction